android/src/main/java/com/heapanalytics/reactnative/RNHeapLibraryModule.java

```diff
@@ -28,18 +27,7 @@ public String getName() {
 
   @ReactMethod
   public void setAppId(String appId) {
-    try {
-      Field skipInstrumentorCheckField = HeapImpl.class.getDeclaredField("skipInstrumentorChecks");
-      skipInstrumentorCheckField.setAccessible(true);
-      skipInstrumentorCheckField.setBoolean(null, true);
-    } catch (NoSuchFieldException e) {
-      android.util.Log.e("InstrumentorCheck",
-              "Caught NoSuchFieldException when trying to skip the instrumentor checks", e);
-    } catch (IllegalAccessException e) {
-      android.util.Log.e("InstrumentorCheck",
-              "Caught IllegalAccessException when trying to skip the instrumentor checks", e);
-    }
-
+    HeapImpl.skipInstrumentorChecks = true;
```

Comment:
Is this temporary? It's awkward to import `HeapImpl`.  Is the plan to expose an interface to skip the checks on the `Heap` object, or to simply remove this when we support autotrack?

Reply:
Yeah, agreed that this is awkward.  We will certainly need to contend with some bigger questions about the interplay between the existing Android lib and the bridge down the road.  I will add a comment before merging.